sdk/appconfiguration/Azure.Data.AppConfiguration/src/CustomHeadersPolicy.cs

```diff
@@ -1,53 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using Azure.Core;
-using Azure.Core.Pipeline;
-
-namespace Azure.Data.AppConfiguration
-{
-    internal class CustomHeadersPolicy : HttpPipelineSynchronousPolicy
```

Comment:
it's been here for 4 years, but we never documented this 'feature' allowing to set custom headers.
Perhaps some old experiments that broke with us starting to suppress things. 

We should not user Activities to pass config around.

It seems safe to just remove this policy.